bindings/python/tests/test_storage.py

```diff
@@ -34,6 +34,11 @@
 # Create an instance
 inst = myentity(dimensions=[2, 3], id="my-data")
 inst["a-bool-array"] = True, False
+inst["a-relation"] = "subj", "pred", "obj"
+inst["a-relation-array"] = [
+    ("s1", "p1", "o1"),
+    ("s2", "p2", "o2"),
+]
```

Comment:
Same as above, can you add a test that checks inst?

Reply:
Ok, added some additional assertments. Had to add a `__eq__()` method to Relation for the test to work.